packages/sanity/src/_internal/cli/commands/migration/runMigrationCommand.ts

```diff
@@ -111,6 +116,18 @@ const createMigrationCommand: CliCommandDefinition<CreateFlags> = {
       throw new Error('Can only dry run migrations from a dataset export file')
     }
 
+    let concurrency = args.extOptions.concurrency
+    if (concurrency !== undefined) {
+      if (concurrency > MAX_MUTATION_CONCURRENCY) {
+        concurrency = MAX_MUTATION_CONCURRENCY
```

Comment:
so this is just warning but not a real hard limit right?

Reply:
it is capped at the max value, so you can't go beyond it, but for the cli it will be a warning not an error. might be better to throw an error?

Reply:
might be better for consistency but would changing this in the future be a breaking change?

Reply:
not sure if I follow – what would be a breaking change?

Reply:
If you have don't have an error at MAX, but in future release you add an error is that considered a breaking change?

Reply:
ah, I see now. That's a good point, strictly speaking that would be a breaking error yes. I'll turn it into an error now. Thanks!